src/Api/API.php

```diff
@@ -2518,8 +2518,7 @@ protected static function parseDropdowns($fields, $params = [])
                 }
 
                 if (
-                    !empty($value)
-                    || $key == 'entities_id' && is_integer($value) && $value >= 0
+                    is_integer($value) && $value >= 0
```

Comment:
`value=0` should be accepted only for `entities_id`
```suggestion
                    is_integer($value)
                    && ($value > 0 || ($key === 'entities_id' && $value >= 0))
```